InteractiveHtmlBom/ecad/schema/genericjsonpcbdata_v1.schema

```diff
@@ -602,6 +605,22 @@
             "properties": {
             },
             "title": "ExtraData"
+        },
+        "FontData": {
+            "type": "object",
+            "additionalProperties": {
+                "type": "object",
+                "properties": {
+                    "w": { "type": "number" },
+                    "l": { "$ref": "#/definitions/Polygons" }
```

Comment:
This will confuse people because what these coordinates describe is not polygons, even if data has same format. In this case this is an array of polylines.

Can you try adding a definition PolyLineArray that just references Polygons? I think it should work and will be clearer.